### DISCUSSION TITLE
remove dashboard     -v3

### DISCUSSION THREAD

**Discussion by shebaoting** [filamentphp/filament#12440] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Package
Panel builder

Package Version
v3

How can we help you?
I want the Panel to default to the xxx page when it is opened, not the dashboard. What should I do?

**Comment by dissto** [comment]
Perhaps `homeUrl`?
```php

return $panel
->homeUrl('/x')
```

**Comment by yuvrajtimalsina** [comment, marked as answer]
**You may check it as well :**

[https://laraveldaily.com/post/filament-disable-dashboard-auto-redirect-url-after-login](url)

**Comment by shebaoting** [comment]
How to hide or disable dashboard?

**Reply by yuvrajtimalsina** [reply]
If you delete dashboard entry in AdminPanelProvider's array it will fall back to first navigation item.
